extensions/resource-deployment/src/ui/deployAzureSQLVMWizard/pages/sqlServerSettingsPage.ts

```diff
@@ -0,0 +1,271 @@
+		super(
+			constants.SqlServerSettingsPageTitle,
+			'',
+			wizard
+		);
+
+	}
+
+	public async initialize() {
+		this.pageObject.registerContent(async (view: azdata.ModelView) => {
+
+			await Promise.all([
+				this.createSqlConnectivityDropdown(view),
+				this.createPortText(view),
+				this.createSqlAuthentication(view),
+			]);
+
+
+			// this._sqlStorageOptimiazationDropdown = view.modelBuilder.dropDown().withProperties<azdata.DropDownProperties>({
+			// 	values: [{
+			// 		displayName: 'General',
+			// 		name: 'GENERAL',
+			// 	},
+			// 	{
+			// 		displayName: 'Transactional Processing',
+			// 		name: 'OLTP',
+			// 	},
+			// 	{
+			// 		displayName: 'Data Warehousing',
+			// 		name: 'DW',
+			// 	}]
+			// }).component();
+
+			// this._sqlStorageOptimiazationDropdown.onValueChanged((value) => {
+			// 	this.wizard.model.sqlOptimizationDropdown = (this._sqlStorageOptimiazationDropdown.value as azdata.CategoryValue).name;
+			// });
+
+			// this.wizard.model.sqlOptimizationDropdown = (this._sqlStorageOptimiazationDropdown.value as azdata.CategoryValue).name;
+
+
+			this._form = view.modelBuilder.formContainer()
+				.withFormItems(
+					[
+						{
+							component: this.wizard.createFormRowComponent(view, constants.SqlConnectivityTypeDropdownLabel, '', this._sqlConnectivityDropdown, true)
+						},
+						{
+							component: this._portTextRow
+						},
+						{
+							component: this.wizard.createFormRowComponent(view, constants.SqlEnableSQLAuthenticationLabel, '', this._sqlAuthenticationDropdown, true)
+						},
+						{
+							component: this._sqlAuthenticationTextRow
+						},
+						{
+							component: this._sqlAuthenticationPasswordTextRow
+						},
+						{
+							component: this._sqlAuthenticationPasswordConfirmationTextRow
+						}
+						// {
+						// 	component: this.wizard.createFormRowComponent(view, 'SQL Storage Optimization Type', '', this._sqlStorageOptimiazationDropdown, true)
+
+						// }
+					],
+					{
+						horizontal: false,
+						componentWidth: '100%'
+					})
+				.withLayout({ width: '100%' })
+				.component();
+
+			return view.initializeModel(this._form);
+		});
+	}
+
+	public async onEnter(): Promise<void> {
+
+		this.liveValidation = false;
+
+		this.wizard.wizardObject.registerNavigationValidator(async (pcInfo) => {
+			if (pcInfo.newPage < pcInfo.lastPage) {
+				return true;
+			}
+
+			this.liveValidation = true;
+
+			let showErrorMessage = await this.formValidation();
+
+			if (showErrorMessage !== '') {
+				return false;
+			}
+			return true;
+		});
+	}
+
+	public onLeave(): void {
+		this.wizard.wizardObject.registerNavigationValidator((pcInfo) => {
+			return true;
+		});
+	}
+
+	private createSqlConnectivityDropdown(view: azdata.ModelView) {
+		this._sqlConnectivityDropdown = view.modelBuilder.dropDown().withProperties(<azdata.DropDownProperties>
+			{
+				values: [
+					{
+						name: 'local',
+						displayName: 'Local (inside VM only)'
```

Comment:
I am seeing a lot of unlocalized string, please go through the PR and make sure all resource strings are properly localized.